package.json

```diff
@@ -21,7 +21,8 @@
     }
   ],
   "dependencies": {
-    "grunt": "0.4.5",
-    "grunt-jquery-content": "3.0.0"
+    "grunt": "1.6.1",
```

Comment:
1.5.3 is the last that supports Node 10 (old infra), 1.6 requires Node 16. That's fine but means it'll break the old infra's staging and we shouldn't tag any stable versions on 1.8.x until we're transitioned.

Alternatively, if 1.5.3 works, maybe we can stay pinned to that on old branches. That should work fine on latest Node as well.

Reply:
1.5.3 works, done.